docs/_docs/FAQ.md

```diff
@@ -110,4 +110,16 @@ nginx.ingress.kubernetes.io/proxy-body-size: "100m"
 
 Please consult the [official documentation](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#custom-max-body-size)
 
+#### Policy conditions do not work for some PURLs
+
+Policy condition values are treated as regular expressions.
+
+1. Policy condition values are implicitly treated as substring matches.
+   They must be explicitly anchored with `^` and `$` to make them an exact match.
+2. Characters with special meaning in regular expressions should be escaped with a `\\`.
```

Comment:
I think in actual regex the escape is just `\` 
It would be `\\` in Json because it has to be escaped there again, or am I misunderstanding something?

Reply:
This condition works for me:
```
{
  "subject": "PACKAGE_URL",
  "operator": "NO_MATCH",
  "value": "^pkg:maven/com.mysql/mysql-connector-j@8.0.33\\?type=jar$"
}
```
(Literal value in json is `^pkg:maven/com.mysql/mysql-connector-j@8.0.33\\?type=jar$`)
This doesn't
```
{
  "subject": "PACKAGE_URL",
  "operator": "NO_MATCH",
  "value": "^pkg:maven/com.mysql/mysql-connector-j@8.0.33\\\\?type=jar$"
}
```
(Literal value in json is `^pkg:maven/com.mysql/mysql-connector-j@8.0.33\?type=jar$`)